AsBuiltReport.NetApp.ONTAP.json

```diff
@@ -12,10 +12,69 @@
 
     },
     "InfoLevel": {
-		"_comment_": "Please refer to the AsBuiltReport project contributing guide for information about how to define InfoLevels.",
-        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive"
+        "_comment_": "0 = Disabled, 1 = Enabled / Summary, 2 = Adv Summary, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive",
```

Comment:
update this to the following for now or until you add more InfoLevels
`"_comment_": "0 = Disabled, 1 = Enabled`

Reply:
Done!